### PR TITLE
Annular flow benchmark

### PR DESCRIPTION
**Pull request by gassmoeller**
Adding a new benchmark case that we used in a submitted paper.

### CODE REVIEW

File: benchmarks/time_dependent_annulus/plugin/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ASPECT; see the file LICENSE.  If not see
+# <http://www.gnu.org/licenses/>.
+
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.8)
+
+FIND_PACKAGE(Aspect 1.5.0 QUIET HINTS ${Aspect_DIR} ../ ../../ $ENV{ASPECT_DIR})
```
Comment (tjhei) [inline benchmarks/time_dependent_annulus/plugin/CMakeLists.txt]: does this work with 1.5?

File: benchmarks/time_dependent_annulus/plugin/time_dependent_annulus.cc [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#include "time_dependent_annulus.h"
```
Comment (tjhei) [inline benchmarks/time_dependent_annulus/plugin/time_dependent_annulus.cc]: header

File: benchmarks/time_dependent_annulus/plugin/time_dependent_annulus.h [hunk 3]
```diff
@@ -0,0 +1,372 @@
+#include <aspect/material_model/simple.h>
```
Comment (tjhei) [inline benchmarks/time_dependent_annulus/plugin/time_dependent_annulus.h]: header

File: benchmarks/time_dependent_annulus/run_all_models [hunk 4]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
```
Comment (tjhei) [inline benchmarks/time_dependent_annulus/run_all_models]: see readme

File: benchmarks/solkz/compositional_fields/solkz_particles.prm [hunk 5]
```diff
@@ -101,7 +101,7 @@ subsection Postprocess
     set Data output format = vtu
     set List of particle properties = function
     set Integration scheme = rk2
-    set Interpolation scheme = cell average
+    set Interpolation scheme = bilinear least squares
```
Comment (tjhei) [inline benchmarks/solkz/compositional_fields/solkz_particles.prm]: this change seems unrelated?

**Comment by gassmoeller** [comment]
I think I addressed all comments.

**Comment by tjhei** [comment]
thanks, ready to be merged!

**Comment by gassmoeller** [comment]
Ok, ready when the testers are. Do not be confused by my last commit, I just squashed and made the test a bit smaller.
